images/linux/scripts/helpers/github-api.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+################################################################################
+##  File:  github-api.sh
+##  Desc:  Helper functions for github api
+################################################################################
+
+function getLatestReleaseURI {
+     if [[ $# -eq 0 || "x$1" = "x-h" || "x$1" = "x--help" ]];then
+        echo "usage: " >&2
+        echo "       $0 organisation/repo format" >&2
+        echo "       $0 organisation repo format" >&2
+        echo >&2
+        echo " Format should be .deb, .tar.gz or other download file extension" >&2
+        exit 1
+    fi
+
+    if [[ "x$1" == *"/"* ]]; then
```

Comment:
Do we really need to support both ways?
I think supporting only first way `url=$1, format=$2` should be enough

Reply:
I think yes. It's annoying to remember suchs small and insignificant  differences. The cost of the mistake in a call is few hours of test unstable build while the cost of the handling both cases close to zero